docs/manuals/long-jobs.qmd

```diff
@@ -34,7 +38,35 @@ cd <your storage volume name>
 ...
 ```
 
-### Step 3: Run your script as background process using `nohup`
+### Step 3: Run your script as background process
+
+We outline two ways of running your long job as a background process:
+
+- Using the UU-provided `run_and_pause` script
+  * the main advantage of this method is that it will automatically [pause your workspace](../responsible-use.qmd#pausing-or-deleting-workspaces) when the job is finished, thereby saving credits and resources.
+  * the downsides are that the script is only available on [UU-maintained Catalog Items](../workspace-catalogue.qmd), and that it requires a ResearchCloud API token.
+- Using the generic method `nohup`
+
+#### Method 1: using `run_and_pause`
+
+*Additional prerequisites*:
+
+- a personal ResearchCloud API token. See [here](https://servicedesk.surf.nl/wiki/spaces/WIKI/pages/117178402/SRC+API) for how to acquire one.
+  * **NOTE**: an API token is sensitive information. Treat it safely, just as you would treat a password!
+- a workspace that contains the `run_and_pause` script. You can check whether it is installed by just typing `run_and_pause` in your terminal. All [UU-maintained catalog items](../workspace-catalogue.qmd) should have the script installed.
+
+To run your long job and automatically pause the workspace when it is finished, simply perform the following steps:
+
+1. Run `run_and_pause "mycommand myarg1 myarg2"` in your terminal.
+1. The script will prompt you for your ResearchCloud API token. Enter it.
+1. The script will explain that your process will be run in a new [tmux](https://github.com/tmux/tmux/wiki) session. `tmux` is a program that allows you to run a process, detach from it (that is: stop seeing its output), and reconnect to it later (start seeing its output again). The script will tell you which command you can use later to reattach to the `tmux` session -- note it down for convenience.
+1. Press any key to start your process in `tmux`.
+
+Your job will start running in new `tmux` "window". You can detach from this window, and go back to your original terminal, using the key combination `Control-b d` (first press `control` and `b` together, then release, then press `d`). Since your process is now running inside `tmux`, you can safely exit your terminal and reconnect at any time.
```

Comment:
```suggestion
Your job will start running in a new `tmux` "window". You can detach from this window, and go back to your original terminal, using the key combination `Control-b d` (first press `control` and `b` together, then release, then press `d`). Since your process is now running inside `tmux`, you can safely exit your terminal and reconnect at any time.
```